superset-frontend/plugins/plugin-chart-echarts/src/BigNumber/BigNumberViz.tsx

```diff
@@ -159,13 +165,25 @@ class BigNumberVis extends React.PureComponent<BigNumberVisProps> {
     });
     container.remove();
 
+    const onContextMenu = (e: MouseEvent<HTMLDivElement>) => {
+      if (this.props.onContextMenu) {
+        e.preventDefault();
+        this.props.onContextMenu(
+          [],
```

Comment:
For this one, even though you can't drill to detail based on the trendline because the timestamp is not a dimension (I'm assuming?), it was a little confusing for me to not be able to right-click on the trendline at all.  What would you think about adding a plain "Drill to detail" menu option that doesn't add any filters when you right-click on the trendline?

Reply:
Added right-click to the trendline

https://user-images.githubusercontent.com/70410625/183649718-96cb22cf-5547-4f47-bd59-d5d598d59940.mov